src/nunavut/lang/c/support/serialization.j2

```diff
@@ -27,6 +27,7 @@ extern "C"
 {% endif -%}
 #include <stdbool.h>
 #include <stdint.h>
+#include <assert.h>
```

Comment:
Please remove. assert.h must not be required. Many embedded platforms define custom assertion logic and do not use `assert`. If this becomes onerous we can add a "custom include list" option in a later review that allows users to specify a list of additional include statements to generate with each datatype.